### PR TITLE
i18n(es): Translated `captcha.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `captcha.mdx`

### CODE REVIEW

File: src/content/docs/es/recipes/captcha.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: Verificar un Captcha
+description: Learn how to create an API route and fetch it from the client.
```
Comment (dreyfus92) [inline src/content/docs/es/recipes/captcha.mdx:3]:
```suggestion
description: Aprende como crear una ruta de API y consultarla desde el cliente.
```

File: src/content/docs/es/recipes/captcha.mdx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        <script is:inline>
+          function onSubmit(token) {
+            fetch("/recaptcha", {
+              method: "POST",
+              body: JSON.stringify({ recaptcha: token })
+            })
+            .then((response) => response.json())
+            .then((gResponse) => {
+              if (gResponse.success) {
+                // Captcha verification was a success
```
Comment (dreyfus92) [inline src/content/docs/es/recipes/captcha.mdx:63]:
```suggestion
                // Verificación del Captcha fue un éxito
```

File: src/content/docs/es/recipes/captcha.mdx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+              } else {
+                // Captcha verification failed
```
Comment (dreyfus92) [inline src/content/docs/es/recipes/captcha.mdx:65]:
```suggestion
                // Falló la verificación del Captcha
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Te dejo unas sugerencias @Waxer59, buen trabajo 😁

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**